### PR TITLE
refactored READMEs moved most to docs

### PR DESCRIPTION
**Pull request by henriterhofte**

### CODE REVIEW

File: docs/documenting/device-info.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Twomes Device Info page
+
+The purpose of a device info page is allow subjects and other people that encounter your measurement device is to find more information what the device is doing easily. The contents of the folder [/docs/deviceinfo/](/docs/deviceinfo/) of a Twomes measurement device repository can be used as the template for this. On a [M5Stack CoreInk](https://github.com/m5stack/M5-CoreInk) device, after the device was provisioned, the e-ink screen can show a QR-code that leads users directy to the device info page.  
+
+## Prerequisites
+You only need a web browser and a plain text editor (or your favourite IDE).
+
+## Writing
+Adapt the content of `index.html` to fit the needs of your measurement device type. Be sure to write in the language the intended audience of the subjects and other pople that might encounter your measurement device. The 
+.[Twomes provisioning app](https://github.com/energietransitie/twomes-app-warmtewachter). After scanning its QR-code, the manual of your device type is presented to the user by the [Twomes provisioning app](https://github.com/energietransitie/twomes-app-warmtewachter).
```
Comment (n-vr) [inline docs/documenting/device-info.md:10]: @henriterhofte This paragraph has some incorrect sentences, but I am not sure as to what they should be.

File: docs/documenting/installation-manual.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Twomes Measurement Device Installation Manual
+
+The purpose of a device installation manual is to help users physically install and power up devices of your specific measurement device type. The contents of the folder [/docs/installationmanual/](/docs/installationmanual/) of a Twomes measurement device repository can be used as a template for this.
```
Comment (n-vr) [inline docs/documenting/installation-manual.md:3]: > The contents of the folder [/docs/installationmanual/](/docs/installationmanual/) of a Twomes measurement device repository can be used as a template for this.

@henriterhofte Do you want to link to a specific file in a repository? If not, I suggest changing this to the following:
```suggestion
The purpose of a device installation manual is to help users physically install and power up devices of your specific measurement device type. The contents of the folder `/docs/installationmanual/` of a Twomes measurement device repository can be used as a template for this. 
```

File: docs/documenting/device-info.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* `div.text-block` with predefined styling for texts.
+
+## Previewing
+Open the file `preview.html` using a regular web browser to preview your manual as shown in the Twomes app. To make the preview resemble the size and aspect ratio of a smartphone screen:
+1. Press the `Ctrl + Shift + I` keys (or `F12` ) on your keyboard to access the Developer Tools.
+2. Press the `Ctrl + Shift + M` keys on to enter (mobile) device emulation mode.
+
+By default, your preview is now in 'Responsive' mode. Using the dropdown menu (above the preview), you can also select custom dimensions for specific devices, such as the iPhone 6/7/8.
+
+![Image](../deviceinfo/device.jpg)
```
Comment (n-vr) [inline docs/documenting/device-info.md:26]: I am not sure what this picture adds to this documentation?

File: docs/documenting/installation-manual.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+* `div.text-block` with predefined styling for texts.
+
+## Previewing
+Open the file `preview.html` using a regular web browser to preview your manual as shown in the Twomes app. To make the preview resemble the size and aspect ratio of a smartphone screen:
+1. Press the `Ctrl + Shift + I` keys (or `F12` ) on your keyboard to access the Developer Tools.
+2. Press the `Ctrl + Shift + M` keys on to enter (mobile) device emulation mode.
+
+By default, your preview is now in 'Responsive' mode. Using the dropdown menu (above the preview), you can also select custom dimensions for specific devices, such as the iPhone 6/7/8.
+
+![Image](../installationmanual/device.jpg)
```
Comment (n-vr) [inline docs/documenting/installation-manual.md:25]: I am not sure what this picture adds to this documentation?

File: docs/releasing/releasing.md [hunk 5]
```diff
@@ -1,16 +1,17 @@
 # Release generic firmware for Twomes measurement devices
```
Comment (n-vr) [inline docs/releasing/releasing.md:1]: @henriterhofte This information is very specific to us. Do we need to make this more universal?:
- You need a workflow file (link to it)
- Someone with the permissions to create a release has to publish it.

File: docs/releasing/releasing.md [hunk 6]
```diff
@@ -36,7 +37,7 @@ Follow the steps below to start the GitHub actions workflow which build the bina
     ```
 
 ## Verifying a draft release
-Only a member of the research group logged into GitHub can see draft releases.
+Only a member of the [research group Energy Transition](https://github.com/energietransitie) logged into GitHub can see draft release.
```
Comment (n-vr) [inline docs/releasing/releasing.md:40]: See previous comment.

File: test/generic_test/nvs.csv [hunk 7]
```diff
@@ -1,3 +1,2 @@
 key,type,encoding,value
 twomes_storage,namespace,,
-mac_addresses,data,string,ab:cd:ef:01:23:45
```
Comment (n-vr) [inline test/generic_test/nvs.csv:3]: @henriterhofte is it not useful to leave an example of how to use this file? Maybe even add a second one to show how to use the `;` to separate them.

**Review by n-vr — CHANGES_REQUESTED** [review]
Please see my comments.

File: docs/documenting/device-info.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
+# Twomes Device Info page
+
+The purpose of a device info page is allow subjects and other people that encounter your measurement device is to find more information what the device is doing easily. The contents of the folder [/docs/deviceinfo/](/docs/deviceinfo/) of a Twomes measurement device repository can be used as the template for this. On a [M5Stack CoreInk](https://github.com/m5stack/M5-CoreInk) device, after the device was provisioned, the e-ink screen can show a QR-code that leads users directy to the device info page.
```
[the remark this answers is not in the document]
Comment (henriterhofte) [reply docs/documenting/device-info.md]: fixed in https://github.com/energietransitie/twomes-generic-esp-firmware/pull/61/commits/9f0fba3142314d61b2eb14cf0c75a22ecd945cb5

File: docs/documenting/device-info.md (hunk 1 above)
Comment (henriterhofte) [reply docs/documenting/device-info.md]: fixed n https://github.com/energietransitie/twomes-generic-esp-firmware/pull/61/commits/9f0fba3142314d61b2eb14cf0c75a22ecd945cb5

File: docs/documenting/device-info.md (hunk 3 above)
Comment (henriterhofte) [reply docs/documenting/device-info.md]: fixed in https://github.com/energietransitie/twomes-generic-esp-firmware/pull/61/commits/9f0fba3142314d61b2eb14cf0c75a22ecd945cb5

File: docs/documenting/installation-manual.md (hunk 4 above)
Comment (henriterhofte) [reply docs/documenting/installation-manual.md]: fixed in https://github.com/energietransitie/twomes-generic-esp-firmware/pull/61/commits/9f0fba3142314d61b2eb14cf0c75a22ecd945cb5

File: docs/releasing/releasing.md (hunk 5 above)
Comment (henriterhofte) [reply docs/releasing/releasing.md]: I tried to make it a bit more universal in https://github.com/energietransitie/twomes-generic-esp-firmware/pull/61/commits/9f0fba3142314d61b2eb14cf0c75a22ecd945cb5

I think there might still be an overlap in docs\releasing\releasing.md versus docs\releasing\secure_boot_firmware.md

Can you check whether the sections starting at /docs/releasing/secure_boot_firmware.md#L10 and /docs/releasing/secure_boot_firmware.md#L16 are still needed? It seems the GitHub Actions workflow (https://github.com/energietransitie/twomes-generic-esp-firmware/blob/main/.github/workflows/release.yml which seems also documented in docs\releasing\releasing.md is all we need?

File: docs/releasing/releasing.md (hunk 6 above)
Comment (henriterhofte) [reply docs/releasing/releasing.md]: fixed in https://github.com/energietransitie/twomes-generic-esp-firmware/commit/9f0fba3142314d61b2eb14cf0c75a22ecd945cb5

**Review by n-vr — APPROVED** [review]
I think it looks good.

**Comment by n-vr** [comment]
The workflows should now run.

**Comment by n-vr** [comment]
Fixing workflows will be done in another PR

**Pull request merged**